src/components/button/partial-styles/_has-reduced-presence.scss

```diff
@@ -34,7 +34,8 @@ $speed-of-reducing-presence: 0.3s;
             }
             limel-icon,
             limel-spinner,
-            svg {
+            svg,
+            .mdc-button__icon {
```

Comment:
Looks good and safe, but I can't reproduce it, presumably because all the icons in the docs examples are svgs, so are covered by the old selector anyway? Where or how did you discover this was a problem? 😄

Reply:
I wanted to work on our action buttons slot, mainly to improve consulters' experience and UI of this area, and faced with some empty divs even if some actions (like reset or save) are not presented. It's here:

<img width="1440" alt="Screenshot 2023-03-29 at 09 59 06" src="https://user-images.githubusercontent.com/50618208/228466506-1784eb94-5ab5-4b2a-bc9a-18f42dc1acfc.png">

Reply:
Ah! I see! Nice! 👍